example/src/index.ts

```diff
@@ -1,13 +1,24 @@
-import { Data, FlameChart } from '../../src';
```

Comment:
Please revert changes to this file. I'll add your plugin to the example myself.)

Reply:
.... my ability to test is using this code. Would you like to take ownership of this PR and change it to your liking?

Reply:
reverted